### PR TITLE
feat: add multi-arch build

### PR DESCRIPTION
**Pull request by darkweaver87**
This PR adds a multi-arch build to the project. You can build and push to your repository this way:
```bash
make IMAGE_REPOSITORY_NAME=myaccount/eventrouter:vx.x.x image-push
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,15 +14,15 @@
 
 TARGET = eventrouter
 GOTARGET = github.com/openshift/$(TARGET)
-LOCAL_IMAGE_TAG=openshift/logging-eventrouter
-IMAGE_REPOSITORY_NAME=quay.io/openshift/logging-eventrouter:latest
+IMAGE_REPOSITORY_NAME ?= quay.io/openshift/logging-eventrouter:latest
```
Comment (jcantrill) [inline Makefile:17]: please change this to "openshift-logging"

File: Makefile [hunk 2]
```diff
@@ -32,7 +32,12 @@ fmt:
 	@echo gofmt
 
 image:
-	podman build  -f Dockerfile.rhel8 -t $(LOCAL_IMAGE_TAG) .
+	buildah build --platform=$(TARGET_PLATFORMS) --manifest eventrouter -f Dockerfile.rhel8
```
Comment (jcantrill) [inline Makefile:35]: why switch to buildah directly instead of podman?

File: Makefile [hunk 3]
```diff
@@ -32,7 +32,12 @@ fmt:
...
+	buildah tag localhost/eventrouter $(IMAGE_REPOSITORY_NAME)
+
+image-push: image
+	podman manifest push --all $(IMAGE_REPOSITORY_NAME) docker://$(IMAGE_REPOSITORY_NAME)
```
Comment (jcantrill) [inline Makefile:39]: I'm unfamiliar here but why do we need the "docker" prefix given podman does not require a docker daemon?

File: Makefile (hunk 1 above)
Comment (darkweaver87) [reply Makefile]: done

File: Makefile (hunk 2 above)
Comment (darkweaver87) [reply Makefile:35]: Actually I was wondering why the project changed from docker to podman few months ago 😂
That's the suggested way of podman: https://podman.io/blogs/2021/10/11/multiarch.html

podman can be launched with all the architectures as a parameter but the manifest will contain only the last processed. It's as you wish, I can do a for loop, invoking podman for earch architecture and add to the manifest using `podman manifest`.

Let me know what you prefer :-)

File: Makefile (hunk 3 above)
Comment (darkweaver87) [reply Makefile:39]: The destination can be a directory or other things and here we (at least I) want to push to an image in a registry implementing the Docker Registry HTTP API V2.

**Pull request merged**